tsdb/head_test.go

```diff
@@ -5399,3 +5399,26 @@ func TestCuttingNewHeadChunks(t *testing.T) {
 		})
 	}
 }
+
+func TestPostingsCardinalityStats(t *testing.T) {
+	head := &Head{postings: index.NewMemPostings()}
+	head.postings.Add(1, labels.FromStrings(labels.MetricName, "t", "n", "v1"))
+	head.postings.Add(2, labels.FromStrings(labels.MetricName, "t", "n", "v2"))
+
+	statsForMetricName := head.PostingsCardinalityStats(labels.MetricName, 10)
+	// Using cache.
```

Comment:
What does this comment refer to?

Reply:
This means that the cache is used within the valid duration, and adding a new series does not change the result. I have moved this down by one line.